dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -312,6 +311,49 @@ else
     exit 1
 fi
 
+# -------------------------------------------------------------
+# create CHE service and route
+# -------------------------------------------------------------
+echo "[CHE] Creating serviceaccount, service and route for CHE pod"
+ echo "apiVersion: v1
```

Comment:
it looks strange to have yaml content in a shell script and not in its own yml file

Reply:
we have a lot of that already, but yeah a I completely agree with that. Some times it is need to be able to use ENV vars for that yaml

Reply:
but there are some sed commands for that no ?

Reply:
no, no sed at all.
https://github.com/eclipse/che/blob/master/dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh#L391-L400
https://github.com/eclipse/che/blob/master/dockerfiles/init/modules/openshift/files/scripts/multi-user/deploy_postgres_and_keycloak.sh#L25-L38
https://github.com/eclipse/che/blob/master/dockerfiles/init/modules/openshift/files/scripts/multi-user/deploy_postgres_only.sh

we have a lot of that, I'm not saying that this is good, but nothing new.

Reply:
I think that the true way is to put everything into one deployment config, but since we are not there I'm OK to do this approach.
@riuvshin please, consider reworking everything to deployment config file(s) in the further improvements.

Reply:
yeah, sigle DC that is definitely a proper way, but yes we are far from there and we will go that way.

Reply:
@benoitf WDYT?